packages/map-template/src/components/LocationsList/LocationsList.jsx

```diff
@@ -21,14 +25,16 @@ function LocationsList({ onBack, onLocationClick, locations }) {
                     <CloseIcon />
                 </button>
             </div>
-            <div className="locations-list__list">
-                {locations?.map(location =>
-                    <ListItemLocation
-                        key={location.id}
-                        location={location}
-                        locationClicked={e => onLocationClick(e)}
-                    />
-                )}
+            <div className="locations-list__scrollable prevent-scroll" {...scrollableContentSwipePrevent}>
```

Comment:
I wonder if this will make the list unscrollable when it should be scrollable: when the sheet is in 100% height and there are more locations than can be shown.

Reply:
As far as I tested it, when the sheet is in 100% height (max snap point), the list is scrollable even if we have a long list.

Reply:
👏